.angular-cli.json

```diff
@@ -9,7 +9,8 @@
       "outDir": "dist/build",
       "assets": [
         "assets",
-        "favicon.ico"
+        "lKiKd11I2I",
+        "sitemap.xml"
```

Comment:
What is this gibberish?

Reply:
Look into the changed files list. lKiKd11I2I/Poradnik.pdf
It's stored in the directory with funny random name because we don't want it to be available for anyone but people to whom our VPM (@kopanska) sends the link.

Reply:
BTW dlaczego rozmawiamy po angielsku?

Reply:
I see but what happened to `favicon.ico`?

Reply:
favicon.ico is under assets directory anyway so this line is just not necessary